packages/lumx-react/src/components/autocomplete/__mockData__/index.ts

```diff
@@ -1,3 +1,4 @@
+/* tslint:disable object-literal-sort-keys */
```

Comment:
We could remove this rule for good :man_shrugging:

Reply:
I'd say we can do that in another PR in which we remove all old rules, remove tslint and install latest version of eslint with the new rules from lumapps-web

Reply:
Btw we could extract rules from lumapps-web and create a separate package we could reuse in every web repo